vllm/worker/model_runner.py

```diff
@@ -593,25 +648,17 @@ def _prepare_model_input(
                 num_kv_heads=self.model_config.get_num_kv_heads(
                     self.parallel_config),
                 head_dim=self.model_config.get_head_size(),
-                page_size=16,
+                page_size=self.block_size,
```

Comment:
is it compatible to all page size?

Reply:
yes, it has not constraint on page size.